rollup.config.js

```diff
@@ -27,6 +28,11 @@ export default {
       css: (css) => {
         css.write('public/static/bundle.css');
       },
+      include: ['src/**/*.svelte', 'node_modules/@graph-paper/**/*.svelte'],
```

Comment:
Why is this line required? Shouldn't this work as-is?

Reply:
Note to self: I was pretty sure that it was, but this component library doesn't require this:

https://github.com/hperrin/smui-example-rollup/blob/master/rollup.config.js

Reply:
You're right. I was playing around with a bunch of different settings and mistakenly thought this was one of the changes that was relevant, but it isn't. The PostCSS stuff is still needed, though.